index.html

```diff
@@ -45,7 +45,7 @@
     edDraftURI: "https://w3c-ccg.github.io/vc-api/",
 
     // subtitle for the spec
-    subtitle: "A HTTP API for Verifiable Credentials lifecycle management",
+    subtitle: "A API for Verifiable Credentials lifecycle management",
```

Comment:
```suggestion
    subtitle: "An HTTP API for Verifiable Credentials lifecycle management",
```

This is the only instance that I think we should keep.